react/features/toolbox/components/web/Toolbox.js

```diff
@@ -633,6 +626,19 @@ class Toolbox extends Component<Props, State> {
         this._doToggleScreenshare();
     }
 
+    _onToolbarHelpPressed: () => void;
+
+    /**
+     * Opens a new window with the user documentation.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onToolbarHelpPressed() {
+        sendAnalytics(createToolbarEvent('help.pressed'));
+        window.open(this.props._helpLink);
```

Comment:
Do you need noopener for safety?

Reply:
what is noopener?

Reply:
Do you mean to check if the _helpLink is available before calling `window.open`?